sbi/utils/conditional_density_utils.py

```diff
@@ -294,7 +294,7 @@ def __init__(
         self.device = self.potential_fn.device
         self.allow_iid_x = allow_iid_x
 
-    def __call__(self, theta: Tensor, track_gradients: bool = True) -> Tensor:
+    def __call__(self, theta: Tensor, track_gradients: bool = True, x_o=None) -> Tensor:
```

Comment:
I know this is the fix for the new `CallablePotential`, see #984 , but not sure that's what we want. @michaeldeistler should have a look

Reply:
also, maybe move the arg to second position and add a docstring entry for it.